.github/workflows/ci.yml

```diff
@@ -43,7 +43,8 @@ jobs:
     - run: cargo build --no-default-features --features prost-codec
     - run: cd proto && cargo build --no-default-features --features prost-codec
     - run: cargo build
-    - run: cargo test --all
+      # Some of the bindgen tests generate warnings, see https://github.com/rust-lang/rust-bindgen/issues/1651
+    - run: RUSTFLAGS="" cargo test --all
```

Comment:
How about adding `-A deref_nullptr` to `RUSTFLAGS`? So that other warnings can still be checked.

Reply:
Ah yes good point, will do that 👌